### DISCUSSION TITLE
Dark / Light Mode image

### DISCUSSION THREAD

**Discussion by juliankasimir** [microsoft/vscode-discussions#1237] [Extension Development QnA] [answered]
We have a picture element in Github where we display the dark and light mode images.
However, this is not displayed correctly in the [store](https://marketplace.visualstudio.com/items?itemName=jolution.todo-nukem-snippet-vscode).

I mean that SVG wasn't allowed either. How do you deal with it ?

The readme is an image from GitHub where there is dark/light mode, so I wouldn't like to commit to just one.

How do you do this?

![darklightmode](https://github.com/microsoft/vscode-discussions/assets/120172350/2195c3c9-fb37-42b8-a1fd-cbaca5ee3825)

Code:

```
<p align="center">
    <picture>
        <source media="(prefers-color-scheme: dark)" srcset="./images/todonukem-dark.png">
        <source media="(prefers-color-scheme: light)" srcset="./images/todonukem-light.png">
        <img alt="Shows the banner of TODO NUKEM, with its logo" src="./images/todonukem-dark.png" width="700">
    </picture>
</p>
```

**Comment by tamuratak** [comment, marked as answer]
You had better report to https://github.com/microsoft/vsmarketplace/issues.

A related request: https://github.com/microsoft/vsmarketplace/issues/281

**Comment by juliankasimir** [comment]
@tamuratak Thanks, i reported it now there:
https://github.com/microsoft/vsmarketplace/issues/933
